pages/ksphere/konvoy/latest/download/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: layout.pug
+navigationTitle: Download
+title: Download Konvoy
+menuWeight: 2
+excerpt: Learn how to download an instance of Konvoy 
+enterprise: false
+---
+
+When you are ready to download an instance of Konvoy, there are 2 possible scenarios:
+
+1. You have already spoken to your sales representative and requested a download link. See <a href="#Downloading_Email">Downloading Konvoy via Email Link</a> for intructions. 
+2. You want to try Konvoy and are interested in downloading a trial instance of the product. See  <a href="#Downloading_Email">Downloading Konvoy via the Trial Page</a> for instructions. 
+
+## <a name="Downloading_Email">Downloading Konvoy via Email Link</a>
+
+After you contact your sales representative, an automated email from our license server will be sent to your account's primary contact with instructions on how to download Konvoy. Select the correct link within the email to start your download, then follow the instructions in the <a name=/ksphere/konvoy/latest/quick-start>Konvoy Quick Start</a> for your next steps.
```

Comment:
```suggestion
After you contact your sales representative, an automated email from our license server will be sent to your account's primary contact with instructions on how to download Konvoy. Select the link corresponding to the OS that you will be using within the email to start your download, then follow the instructions in the [Konvoy Quick Start](../quick-start/) for your next steps.
```
Konvoy has a link checker as part of our CI, `/ksphere/konvoy/latest/` is not correct in the context of that repo. This is also consistent with all the other links.